gc/base/MemoryPoolAddressOrderedListBase.hpp

```diff
@@ -48,6 +48,7 @@ class MM_MemoryPoolAddressOrderedListBase : public MM_MemoryPool
 	MM_HeapLinkedFreeHeader ** _referenceHeapFreeList;
 	MM_SweepPoolState* _sweepPoolState;	/**< GC Sweep Pool State */
 	MM_SweepPoolManagerAddressOrderedListBase* _sweepPoolManager;		/**< pointer to SweepPoolManager class */
+	MM_HeapLinkedFreeHeader *_lastFreeEntry;							/**< address of the last free entry in the pool; valid after compact; NOT maintained during allocation */ 
```

Comment:
No that is ok.  I agree that _lastFreeEntry really only makes sense on the MPAOL hierarchy.  In the future maybe the compact scheme could be switched so that it knows about the MPAOL hierarchy instead of the top level MP since it only works with MPAOL.  At that point the two APIs on MP could be removed.